### ISSUE TITLE
Reading: On Feature Learning in the Presence of Spurious Correlations

### ISSUE THREAD

**Issue by ayaco0** [ayaco0/paper-survey#8]
### 論文リンク
https://arxiv.org/abs/2210.11369

### 著者/所属機関
Pavel Izmailov, Polina Kirichenko, Nate Gruver, Andrew Gordon Wilson (New York University)

### 投稿日付(yyyy/MM/dd)
2022/08/20

## 概要
-   分類タスクの擬似特徴があるデータセットにおいて，表現学習の質（擬似特徴ではなくコアな特徴をどれだけデコードできているか）に影響する要因について分析
-   擬似特徴：学習したい問題に本質的には関係がないが，学習に使われる特徴のこと
	e.g.画像の前景を分類するタスクにおける画像の背景，テクスチャ
-   表現学習の質にはモデルのアーキテクチャと事前学習手法が影響    
-   強い正則化は表現学習の質を向上させるのに必要なさそう

## 実験
- データセット（それぞれ擬似特徴がある）
  - Waterbirds：鳥の画像の2値分類，水が写ってる=水鳥の擬似特徴
  - CelebA：人の画像の2値分類，ブロンド=女性の擬似特徴
  - FMOW：衛生写真から土地利用のタイプを多値分類，撮影地域の擬似特徴
  - CXR：胸部X線画像の2値分類，胸部ドレーン(?)が擬似特徴
  - MultiNLI：ペアの文章の関係を分類，否定語を含む=矛盾関係の擬似特徴
  - Civil Comments：有害な文章かどうかの2値分類，特定8カテゴリについて言及すれば有害という擬似特徴

- 評価方法
  - DER（Deep feature reweighting ; 最終層の再トレーニング）を用いてどれだけコア特徴がデコードされるかを測定し，表現学習の質を評価
  - 擬似特徴を予測する線形分類器を用いて，学習済み特徴表現から擬似特徴がデコードされる可能性を測定

### ERM vs Group Robustness Training
<img width="504" alt="スクリーンショット 2022-11-25 3 10 35" src="https://user-images.githubusercontent.com/74520178/203846083-69eee05b-3868-43a8-93cd-0bc99a6fd6e1.png">
<img width="236" alt="スクリーンショット 2022-11-25 3 11 42" src="https://user-images.githubusercontent.com/74520178/203846186-4a772abd-2ec6-4018-b66d-4d40de9cc6cb.png">

##### Effect of the model architecture and pretraining
<img width="453" alt="スクリーンショット 2022-11-25 3 12 22" src="https://user-images.githubusercontent.com/74520178/203846251-e97c8354-72f9-49ae-9aef-2829a39de447.png">

##### Effect of training on target data
<img width="460" alt="スクリーンショット 2022-11-25 3 15 38" src="https://user-images.githubusercontent.com/74520178/203846631-164959a7-01da-45f7-9adb-6303b532ad97.png">

##### Effect of pretraining strategy
<img width="226" alt="スクリーンショット 2022-11-25 3 18 28" src="https://user-images.githubusercontent.com/74520178/203846931-29888921-b911-4ac3-b80e-ceafa7aea432.png">

##### Effect of weight decay
<img width="233" alt="スクリーンショット 2022-11-25 3 16 23" src="https://user-images.githubusercontent.com/74520178/203847084-90ff2cde-ccc4-44d8-9804-2ecba0f92a77.png">

##### Effect of Data Augmentation
<img width="216" alt="スクリーンショット 2022-11-25 3 18 35" src="https://user-images.githubusercontent.com/74520178/203846965-8b11f653-4ffb-4d4b-9bbc-4bea7c713c3f.png">

## コメント
- 分析系の論文だから内容だけじゃなくて書き方も参考にできそう
